blockchain/block.go

```diff
@@ -9,6 +9,8 @@ import (
 	"github.com/gitferry/bamboo/types"
 )
 
+var privateKey crypto.PrivateKey
```

Comment:
Why putting `privateKey` here? Please place it within the `crypto` package if possible